source/assets/stylesheets/styles/_container.scss

```diff
@@ -1,24 +1,21 @@
 .container {
   margin: 0 auto;
   max-width: $max-mobile-container;
+  overflow: hidden;
 
   @include breakpoint($tablet-breakpoint) {
     max-width: $desktop-breakpoint;
   }
 }
 
 .container--grid {
-  @extend .container;
-  box-sizing: border-box;
-  overflow: hidden;
-  padding-left: 0;
-
   @include breakpoint($tablet-breakpoint) {
-    max-width: ($desktop-breakpoint + 6px);
     margin: 0 -3px;
   }
 
-  @include breakpoint($desktop-breakpoint) {
-    margin: 0 auto;
+  &::after {
+    content: "";
+    display: block;
+    clear: both;
```

Comment:
We should probably re-use the `clearfix` mixin in `_mixins.scss` here (by using `@include`). I know that the recommendations for clearfix change over the years - if this is a more up-to-date implementation we could also re-write the mixin code...

Reply:
Cool, I wasn't aware we had a mixin for it!